app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt

```diff
@@ -357,7 +357,13 @@ class EnrollmentActivity : ActivityGlobalAbstract(), EnrollmentView {
 
     override fun showDateEditionWarning() {
         val dialog = MaterialAlertDialogBuilder(this, R.style.DhisMaterialDialog)
-            .setMessage(R.string.enrollment_date_edition_warning)
+            .setMessage(
+                resourceManager.formatWithProgramEventLabel(
+                    R.string.enrollment_date_edition_warning_event_label,
+                    presenter.getProgram()?.uid(),
```

Comment:
what do you think about send the program uid in the `showDateEditionWarning` as a parameter instead of using the presenter as a getter?

Reply:
Yes, better way! thanks